metabrainz/templates/api/info.html

```diff
@@ -8,14 +8,14 @@
     <h1 class="page-title">MetaBrainz API</h1>
 
     <em>
-    All endpoints require an access token which you can get from your
-    <a href="{{ url_for('users.profile') }}">profile page</a>.
+    {{ _('All endpoints require an access token which you can get from your
+    <a href="{{ url_for(\'users.profile\') }}">profile page</a>.') }}
```

Comment:
You can use string formatting to inject the URL: http://jinja.pocoo.org/docs/2.9/extensions/#newstyle-gettext.